pkg/apis/kf/v1alpha1/space_types.go

```diff
@@ -95,10 +97,30 @@ type SpaceSpecExecution struct {
 	Env []corev1.EnvVar `json:"env,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// SpaceSpecResourceLimits contains definitions for resource usage limits.
+type SpaceSpecResourceLimits struct {
+	// SpaceQuota holds the k8s ResourceQuota created for the whole space.
+	// For now, only one ResourceQuota per space is supported.
+	// Consider allowing multiple ResourceQuotas when more quota scopes are enabled in k8s
+	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	SpaceQuota corev1.ResourceList
+
+	// ResourceDefaults holds the k8s LimitRange created for the whole space,
+	// which sets default request/limit for resources per pod or container.
+	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	ResourceDefaults []corev1.LimitRangeItem
+}
+
 // SpaceStatus represents information about the status of a Space.
 type SpaceStatus struct {
 	// Pull in the fields from Knative's duckv1beta1 status field.
 	duckv1beta1.Status `json:",inline"`
+
+	Quota corev1.ResourceQuotaStatus `json:"quotaStatus,omitempty"`
```

Comment:
I think the `quotaStatus` tag can just be `quota`

Reply:
good catch, updated